install.sh

```
@@ -0,0 +1,341 @@
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+BLUE='\033[0;34m'
+YELLOW='\033[0;33m'
+BOLD='\033[1m'
+NC='\033[0m' # No Color
+
+# Get repo information dynamically if possible, otherwise use default
+get_repo_info() {
+  if command -v git &> /dev/null && [ -d .git ]; then
+    # Get remote origin URL
+    local remote_url=$(git config --get remote.origin.url)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)